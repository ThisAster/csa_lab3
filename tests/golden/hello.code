[{"opcode": "ld", "arg_type": "immediate", "arg": 0},
 {"opcode": "st", "arg_type": "address", "arg": 1},
 {"opcode": "ld", "arg_type": "immediate", "arg": 2},
 {"opcode": "st", "arg_type": "address", "arg": 0},
 {"opcode": "ld", "arg_type": "address", "arg": 1},
 {"opcode": "sub", "arg_type": "immediate", "arg": 11},
 {"opcode": "jl", "arg_type": "address", "arg": 9},
 {"opcode": "ld", "arg_type": "immediate", "arg": 0},
 {"opcode": "jmp", "arg_type": "address", "arg": 10},
 {"opcode": "ld", "arg_type": "immediate", "arg": 1},
 {"opcode": "je", "arg_type": "address", "arg": 22},
 {"opcode": "ld", "arg_type": "address", "arg": 0},
 {"opcode": "st", "arg_type": "address", "arg": 14},
 {"opcode": "ld", "arg_type": "address", "arg": 1},
 {"opcode": "add", "arg_type": "address", "arg": 14},
 {"opcode": "st", "arg_type": "address", "arg": 15},
 {"opcode": "ld", "arg_type": "indirect_address", "arg": 15},
 {"opcode": "out", "arg_type": "address", "arg": 0},
 {"opcode": "ld", "arg_type": "address", "arg": 1},
 {"opcode": "add", "arg_type": "immediate", "arg": 1},
 {"opcode": "st", "arg_type": "address", "arg": 1},
 {"opcode": "jmp", "arg_type": "address", "arg": 4},
 {"opcode": "hlt", "arg_type": "none", "arg": 0}]