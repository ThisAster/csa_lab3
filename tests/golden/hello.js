var s
var i
i = 0
s = "hello world"
while (i < 11) {
    print(s[i])
    i = i + 1
}
