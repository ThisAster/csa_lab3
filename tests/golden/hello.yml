js: hello.js
code: hello.code
data: hello.data
txt: hello.txt
translator_log: hello-1.log
machine_log: hello-2.log
